.github/workflows/source/test_travis_matrix.sh

```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/env sh
+
+cp .github/workflows/source/travis_matrix.py Regression/
+cd Regression/
+
+# Put the name of all travis tests into a text file
+python prepare_file_travis.py && grep "\[" travis-tests.ini > travis_all_tests.txt
```

Comment:
just make this two lines. the `set -eu -o ...` above will make sure the script fails if any of the commands fails